functable.c

```diff
@@ -423,6 +431,56 @@ Z_INTERNAL uint32_t compare256_stub(const uint8_t *src0, const uint8_t *src1) {
     return functable.compare256(src0, src1);
 }
 
+Z_INTERNAL uint32_t longest_match_stub(deflate_state *const s, Pos cur_match) {
```

Comment:
I definitely didn't generate this much of a delta and what's even weirder is it doesn't show any net lines removed. Is github's diff display here drunk? Going to "view file" does not show an extra function definition of "longest_match_stub"

Reply:
It thinks first copy starts at line 99 and second copy starts here at line 434.

Reply:
Yeah, this is how the interactive rebase decided to resolve the delta.  Ugh, revising history is dangerous.  Fixed now.

Reply:
Doesn't really help that there is slight delay on the web version when it (re-)generates new diffs from the commits... I usually wait until it says the diffs are outdated, then reload...